sipa/static/css/agdsn-bootstrap-social.css

```diff
@@ -0,0 +1,67 @@
+.btn-social-icon {
+    position: relative;
+    padding-left: 44px;
+    text-align: left;
+    write-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    height: 34px;
+    width: 34px;
+    padding: 0;
+
+    background-color: var(--btn-social-bg-color);
+    color: #ffffff;
+
+    border: none;
+}
+.btn-social-icon>:first-child{
+    position: absolute;
+    left: 0;
+    top: 0;
+    bottom: 0;
+    width: 32px;
+    line-height: 34px;
+    font-size: 1.6em;
+    text-align: center;
+    border-right: 1px solid rgba(0, 0, 0, 0.2)
+    border: none;
+    width: 100%;
+}
+
+
+.btn-social-icon:focus,.btn-social-icon.focus{
+    color: #fff;
+    background-color: color-mix(in hsl, black 16%, var(--btn-social-bg-color));
+    border-color: rgba(0, 0, 0, 0.2);
+}
+
+.btn-social-icon:hover {
+    color: #fff;
+    background-color: color-mix(in hsl, black 16%, var(--btn-social-bg-color));
+    border-color: rgba(0, 0, 0, 0.2);
+}
+
+.btn-social-icon:active,.btn-social-icon.active, {
+    color: #fff;
+    background-color: color-mix(in hsl, black 16%, var(--btn-social-bg-color));
+    border-color: rgba(0, 0, 0, 0.2);
+}
+
+.btn-social-icon:active:hover,.btn-social-icon.active:hover,.btn-social-icon:active:focus,.btn-social-icon.active:focus,.btn-social-icon:active.focus,.btn-social-icon.active.focus {
+    color: #fff;
+    background-color: color-mix(in hsl, black, var(--btn-social-bg-color) 20%);
```

Comment:
What is the impact on non-recent browsers? how does it look when `color-mix` is not supported? As we found out it is a rather recent feature after all.

Reply:
Here is the compatibility matrix [0], will try to test with some old enough browser.

[0] https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix#browser_compatibility

Reply:
Ignoring, finding an older browser takes too much time – and: it is the hover rule, anyway.